pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -216,8 +215,8 @@ func getLatestConfig(url string, iteration int,
 				getconfigCtx.configGetStatus = types.ConfigGetTemporaryFail
 			}
 		} else if rtf == types.SenderStatusCertMiss {
-			// trigger a one sec timer to acquire new certs from cloud
-			triggerFetchCerts(getconfigCtx.zedagentCtx)
+			// trigger to acquire new controller certs from cloud
+			triggerControllerCertEvent(getconfigCtx.zedagentCtx)
```

Comment:
As per https://wiki.lfedge.org/display/EVE/Measured+Boot+and+Remote+Attestation, we should be triggering attestation when config fails with 403. Perhaps I am not able to find it but is that already handled? Per my cursory reading of the code, seems like we return SenderStatusNone for anything other than http.StatusOK, http.StatusCreated and http.StatusNotModified?

Reply:
> As per https://wiki.lfedge.org/display/EVE/Measured+Boot+and+Remote+Attestation, we should be triggering attestation when config fails with 403. Perhaps I am not able to find it but is that already handled? Per my cursory reading of the code, seems like we return SenderStatusNone for anything other than http.StatusOK, http.StatusCreated and http.StatusNotModified?

We haven't added support for 403 yet. That requires some new flow where zedagent tells tpmmgr to re-attest, which isn't in place yet.